public/app/routes/routes.tsx

```diff
@@ -309,6 +309,7 @@ export function getAppRoutes(): RouteDescriptor[] {
     // ADMIN
     {
       path: '/admin/authentication',
+      roles: () => contextSrv.evaluatePermission(() => ['Admin'], [AccessControlAction.SettingsRead]),
```

Comment:
Do we need to check both to be consistent with backend? I'm not sure, maybe we want to make it available as a read-only page in future.
```suggestion
      roles: () => contextSrv.evaluatePermission(() => ['Admin'], [AccessControlAction.SettingsRead, AccessControlAction.SettingsWrite]),
```

Reply:
If we do so, we should also align here: https://github.com/grafana/grafana/blob/3d42228737bf15feb463966549ca5d0a0fdd6473/pkg/api/api.go#L228

Reply:
I think that makes sense, I'll add the write permission requirement in both places 👍

Reply:
Oh, looking at the code it looks like `contextSrv.evaluatePermission` returns true if the user has either of the permissions. I think I'll just request the permission to write the settings for now, and assume that if someone has that then they'd also have the permission to read.